doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -9,10 +9,17 @@ Matter OTA
 
 The Matter Over-The-Air is a process that allows a Matter device in a Matter fabric to update its firmware.
 Most of the Matter devices will require such an update at some point during their lifecycle.
+For this reason, the Matter specification requires a method for updating the firmware of the device to be implemented for each device.
```

Comment:
I think this is not logically correct (that is, Matter specification requires DFU because of security reasons, not because most of the Matter devices will require...). We should start from telling that the Matter specification requires that each device implement some method of firmware update and then that Matter Over-the-Air update is one of them (we can even say that we recommend it for accessory devices).

Reply:
Rephrased.